packages/cli/src/lib/bundler/packageDetection.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BackstagePackageJson } from '@backstage/cli-node';
+import { Config } from '@backstage/config';
+import chokidar from 'chokidar';
+import fs from 'fs-extra';
+import { join as joinPath, resolve as resolvePath } from 'path';
+import { paths as cliPaths } from '../paths';
+
+const DETECTED_MODULES_MODULE_NAME = '__backstage-autodetected-plugins__';
+
+interface PackageDetectionConfig {
+  include?: string[];
+  exclude?: string[];
+}
+
+function readPackageDetectionConfig(
+  config: Config,
+): PackageDetectionConfig | undefined {
+  const packages = config.getOptional('app.experimental.packages');
+  if (!packages) {
```

Comment:
```suggestion
  if (packages === undefined) {
```

what if it's a zero :)